partner_autocreate_task/data/ir_cron.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
```

Comment:
@rimbendhaou The `noupdate="0"` (which is the default value so we don nett to declare) is not good for a public module.